src/test/resources/org/jetbrains/research/ml/ast/transformations/ifRedundantLinesRemoval/data/pure/in_4_if_elif_else.py

```diff
@@ -0,0 +1,23 @@
+s = input()
+
+if s:
```

Comment:
A little bit strange that you use `if s:` - `elif s:`. Maybe it does not affect the transformation, but what if it does?